lizmap/modules/view/classes/lizmapMapDockItem.class.php

```diff
@@ -17,10 +17,11 @@ class lizmapMapDockItem
     public $order = 0;
     public $css = '';
     public $js = '';
+    public $jsParams = array();
     public $icon = '';
     public $menuIconClasses = '';
 
-    public function __construct($id, $title, $content, $order = 0, $css = '', $js = '')
+    public function __construct($id, $title, $content, $order = 0, $css = '', $js = '', $jsParams = array())
```

Comment:
Documentation on parameters are missing. 

We should indicate, at least:
- `$js` is an URL of a JS file
- `$jsParams` is an array containing additional attributes for the `<script>` tags, like `type` if not `text/javascript`, or `defer` etc...

Reply:
- `$css` is an URL of a JS file